data/interfaces/default/css/tautulli.css

```
@@ -4610,4 +4610,59 @@ a.donate-with-crypto::after {
 .docker-mount {
     margin: 20px auto 0 auto;
     position: relative;
-}
\ No newline at end of file
+}
+
+/* Ace Editor Styles */
+#message-editor {
+    width: 100%;
+    height: 300px;
+    border: 1px solid #444;
+    border-radius: 4px;
+}
+
+/* Custom CSS to reduce Ace editor gutter width */
+#message-editor .ace_gutter-cell {
+    padding-left: 8px;
+    padding-right: 8px;
+}
+
+.editor-controls {
+    padding: 3px;
+}
+
+.editor-container {
+    border: 1px solid #444;
+    border-radius: 4px;
+}
+
+.editor-control-btn {
+    margin-right: 1px;
+    width: 40px;
+    height: 40px;
+    padding: 8px;
+    border-radius: 4px;
+    transition: all 0.2s ease-in-out;
+}
+
+.editor-control-btn:hover {
+    background-color: #555;
+    border-color: #777;
+    transform: translateY(-1px);
+}
+
+.editor-control-btn i {
+    font-size: 14px;
+}
+
+/* Responsive design for editor controls */
+@media (max-width: 768px) {
+    .editor-controls .text-left {
+        text-align: left !important;
+    }
+    
+    .editor-control-btn {
+        margin-right: 6px;
+        width: 36px;
+        height: 36px;
+    }
+}
```

I adjusted some of the CSS styles.

```suggestion
/* Ace Editor Styles */
#message-editor {
    width: 100%;
    height: 300px;
}
/* Custom CSS to reduce Ace editor gutter width */
#message-editor .ace_gutter-cell {
    padding-left: 8px;
    padding-right: 8px;
}
.editor-controls {
    padding: 3px;
    border-bottom: 1px solid #444; 
}
.editor-container {
    background-color: #282828;
    border: 1px solid #444;
    border-radius: 3px;
}
.editor-control-btn {
    margin-right: 1px;
    width: 32px;
    height: 32px;
    padding: 6px;
}
.editor-control-btn:hover {
    background-color: #555;
    border-color: #777;
}
/* Responsive design for editor controls */
@media (max-width: 768px) {
    .editor-controls .text-left {
        text-align: left !important;
    }
    .editor-control-btn {
        margin-right: 6px;
        width: 32px;
        height: 32px;
    }
}
```